root/frontend/src/components/PreviewCard/PreviewCard.tsx

```diff
@@ -1,38 +1,36 @@
 import React from 'react';
-import { useTheme } from 'react-jss';
 import { BsFillSuitDiamondFill } from 'react-icons/bs';
 import { HiDotsHorizontal } from 'react-icons/hi';
-import useStyles from './PreviewCard.style';
+import '../../styles/PreviewCard.scss';
 
 type PropTypes = {
   title: string;
   exercises: string[];
+  theme: string;
   //  placeholder until logos are figured out
   //  logo: string
 };
 
-function PreviewCard({ title, exercises }: PropTypes) {
-  const theme = useTheme();
-  const classes = useStyles({ theme });
+function PreviewCard({ title, exercises, theme }: PropTypes) {
   const listExercise = exercises.map((exercise) => (
-    <li className={classes.listExerciseItem}>
-      <div className={classes.exerciseItem}>
-        <BsFillSuitDiamondFill className={classes.diamond} />
-        <p className={classes.exercise}>{exercise}</p>
+    <li className="listExerciseItem">
+      <div className="exerciseItem">
+        <BsFillSuitDiamondFill style={{ color: theme }} />
+        <p className="exercise">{exercise}</p>
       </div>
     </li>
   ));
 
   return (
-    <div className={classes.previewCard}>
-      <div className={classes.previewCardHeader}>
-        <div className={classes.profileImage} />
-        <h2 className={classes.previewCardTitle}>{title}</h2>
+    <div className="previewCard" style={{ color: theme }}>
+      <div className="previewCardHeader" style={{ color: theme }}>
+        <div className="profileImage" style={{ background: theme }} />
+        <h2 className="previewCardTitle">{title}</h2>
       </div>
-      <ul className={classes.list}>
+      <ul className="list">
         {listExercise}
       </ul>
-      {(exercises.length > 6) && <HiDotsHorizontal size={20} className={classes.ellipse} />}
+      {(exercises.length > 6) && <HiDotsHorizontal size={20} className="ellipse" />}
```

Comment:
Nitpick: I think we might want to have the ellipse be the same colour as the text rather than the same colour as the card theme. Seems that the `fill` property controls that.